### ISSUE TITLE
Error when building from source on Windows

### ISSUE THREAD

**Issue by pete-adriano-debiase** [openstenoproject/plover#948]
## Summary

Running the `./setup.py bdist_win` development helper command produces the following error:

```
running dist\plover-4.0.0.dev8-win32\data\python.exe -m plover_build_utils.get_pip
sha1 does not match: 35ac67d8b15c308b22e3234c8a6f44a652f2d869 instead of 0ba245c9b8334623dbda5bde2fc8726c453df1b5
sha1 does not match: 35ac67d8b15c308b22e3234c8a6f44a652f2d869 instead of 0ba245c9b8334623dbda5bde2fc8726c453df1b5
sha1 does not match: 35ac67d8b15c308b22e3234c8a6f44a652f2d869 instead of 0ba245c9b8334623dbda5bde2fc8726c453df1b5
Traceback (most recent call last):
  File "runpy.py", line 193, in _run_module_as_main
  File "runpy.py", line 85, in _run_code
  File "C:\plover\plover_build_utils\get_pip.py", line 33, in <module>
    get_pip(sys.argv[1:])
  File "C:\plover\plover_build_utils\get_pip.py", line 14, in get_pip
    '0ba245c9b8334623dbda5bde2fc8726c453df1b5')
  File "C:\plover\plover_build_utils\download.py", line 50, in download
    assert os.path.exists(dst), 'could not successfully retrieve %s' % url
AssertionError: could not successfully retrieve https://bootstrap.pypa.io/get-pip.py
Traceback (most recent call last):
  File "C:\plover\setup.py", line 392, in <module>
    cmdclass=cmdclass,
  File "C:\Python3\lib\site-packages\setuptools\__init__.py", line 129, in setup
    return distutils.core.setup(**attrs)
  File "C:\Python3\lib\distutils\core.py", line 148, in setup
    dist.run_commands()
  File "C:\Python3\lib\distutils\dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "C:\Python3\lib\distutils\dist.py", line 974, in run_command
    cmd_obj.run()
  File "C:\plover\setup.py", line 143, in run
    run(dist_py, '-m', 'plover_build_utils.get_pip')
  File "C:\plover\setup.py", line 141, in run
    subprocess.check_call(args)
  File "C:\Python3\lib\subprocess.py", line 291, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '('dist\\plover-4.0.0.dev8-win32\\data\\python.exe', '-m', 'plover_build_utils.get_pip')' returned non-zero exit status 1.
```

## Reproducing

```
cd C:\
git clone https://github.com/openstenoproject/plover.git
python -m pip install -r requirements.txt
pip3 install --user -e . -r requirements_plugins.txt
./setup.py bdist_win
```

Changing the SHA1 on line 14 of the `plover\plover_build_utils\get_pip.py` file from `0ba245c9b8334623dbda5bde2fc8726c453df1b5` to `35ac67d8b15c308b22e3234c8a6f44a652f2d869` makes it possible to build Plover successfully.

Considered creating a pull request, but it almost seems like too small of a change. Please don't hesitate to let me know if you'd like me to go ahead and do that though - I'd be more than happy to. I'm a huge fan of the project.

## Plover Version

Plover 4.0.0.dev8

## System

Windows 10

**Comment by DanLanglois** [comment]
fix was committed 'Fixup CI': [#952 ](https://github.com/openstenoproject/plover/pull/952)

**Comment by pete-adriano-debiase** [comment]
Confirmed working. Thanks very much!
